src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const clickPromis = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+
+  setTimeout(() => {
+    // eslint-disable-next-line prefer-promise-reject-errors
+    reject('Promise was rejected!');
+  }, 3000);
+});
+
+const div = document.createElement('div');
+
+document.body.append(div);
+
+clickPromis
+  .then((message) => {
+    div.classList.add('message');
+    div.textContent = message;
+  })
+  .catch((errorMessage) => {
+    div.classList.add('message', 'error-message');
```

Comment:
You're adding two classes in the catch block: 'message' and 'error-message'. But in the then block, you're only adding 'message'. If you always want the 'message' class to be added, consider adding it when you create the div, so you don't have to repeat yourself in the then and catch blocks.